website/src/main/tut/docs/rules/Disable.md

```diff
@@ -114,6 +75,31 @@ import scalafix.internal.config._
 ```
 ```tut:passthrough
 println(
-scalafix.website.rule("Disable", DisableConfig.default)
+scalafix.website.config[UnlessInsideBlock]("UnlessInsideBlock")
+)
+println(
+scalafix.website.config[DisableConfig]("DisableConfig")
+)
+println(
+scalafix.website.defaults("DisableConfig", DisableConfig.default)
+)
+println(
+scalafix.website.examples[DisableConfig]("DisableConfig")
 )
 ```
+
+The the example configuration above, Scalafix will report the following warnings:
+```scala
+import scala.util.Try
+
+object Test {
+  val z = 1.asInstanceOf[String] // default mode, not ok
+
+  List(1) + "any2stringadd" // unlessSynthetic mode, not ok
```

Comment:
s/unlessSynthetic/ifSynthetic/

Reply:
Non-executable docs are difficult to maintain, would be nice if we could somehow reuse testkit for docs 🤔